//
// Protocol messages for Plexus overlay RPCs
//

package UniSphere.Protocol;

//
// RPC request message
//
message RpcRequest {
  required uint64 rpc_id = 1;
  required string method = 2;
  required bytes data = 3;
}

//
// RPC response message
//
message RpcResponse {
  required uint64 rpc_id = 1;
  required bytes data = 2;
}
